packages/runfiles/runfiles.ts

```diff
@@ -31,9 +31,11 @@ export class Runfiles {
       this.runfilesDir = path.resolve(_env['RUNFILES_DIR']!);
     } else if (!!_env['RUNFILES']) {
       this.runfilesDir = path.resolve(_env['RUNFILES']!);
+    } else if (!!_env['JS_BINARY__RUNFILES']) {
```

Comment:
Should we put `JS_BINARY__RUNFILES` before `RUNFILES` so they are in priority order? The lower the priority the more likely we are to drop it if things like https://groups.google.com/g/bazel-discuss/c/DsVivJhU7Bw/m/trgF86CqDwAJ move forward?

Reply:
Sounds good. I'll update.